mule-extensions-api/src/main/java/org/mule/runtime/extension/api/runtime/route/Chain.java

```diff
@@ -24,8 +24,10 @@
  * occurs in one processor and it is propagated to the {@link Chain}
  *
  * @since 1.0
+ * @deprecated use {@link org.mule.sdk.api.runtime.route.Chain} instead.
  */
 @NoImplement
+@Deprecated
 public interface Chain {
```

Comment:
shouldn't this one extend the SDK API chain?

Reply:
Given that the SDK API Chain uses the Result class from SDK API, wouldn't the methods be different? How can I overcome this?